kuksa-client/kuksa_client/__main__.py

```diff
@@ -113,9 +176,13 @@ def path_completer(self, text, line, begidx, endidx):
 
                 if 'description' in child:
                     description = child['description']
+                elif 'description' in child[key]:
```

Comment:
Not sure if I follow things here.

We loop on all keys in `childTree`. Then we set `child = childTree[key]`. If that is a dict we look for `child[key]`. Do we expect the same `key` to exist on both levels? I .e no need to do a `if key in child` first?

Reply:
I agree, it looks a bit odd.
Since kuksa-val and databroker return diferent data structures I had to make workaround to cover both cases without making extended changes. I have pushed again so I hope it is a bit clearer.